src/content/chainlink-automation/tutorials/dynamic-nft-qs.mdx

```diff
@@ -0,0 +1,106 @@
+---
+section: automation
+date: Last Modified
+title: Creating Dynamic NFTs"
+whatsnext: { "FAQs": "/chainlink-automation/faqs" }
+---
+
+import { Aside, CodeSample, ClickToZoom } from "@components"
+import { YouTube } from "@astro-community/astro-embed-youtube"
+
+Learn how to create NFTs that can change over time, based on user interaction, or any number of conditions by following along with this video tutorial:
+
+<YouTube id="https://www.youtube.com/watch?v=E7Rm1LUKhj4" />
+
+{/* prettier-ignore */}
+<div class="remix-callout">
+  <a href="https://github.com/smartcontractkit/smart-contract-examples/tree/main/dynamic-nft">See the code on GitHub</a>
+</div>
+
+<Aside type="caution" title="Disclaimer">
+  This tutorial represents an example of using a Chainlink product or service and is provided to help you understand how
+  to interact with Chainlink’s systems and services so that you can integrate them into your own. This template is
+  provided “AS IS” and “AS AVAILABLE” without warranties of any kind, has not been audited, and may be missing key
+  checks or error handling to make the usage of the product more clear. Do not use the code in this example in a
+  production environment without completing your own audits and application of best practices. Neither Chainlink Labs,
+  the Chainlink Foundation, nor Chainlink node operators are responsible for unintended outputs that are generated due
+  to errors in code.
+</Aside>
+
+## Overview
+
+In this tutorial, you will use the [Chainlink Automation Network](https://automation.chain.link) to help create a dynamic NFT by using the Automation-compatible smart contract provided in the repository.
+
+The demo includes:
+
+- Setting up and funding an upkeep on the Chainlink Automation Network
+- Setting up a dynamic NFT and viewing the progression of the NFT on [OpenSea](https://testnets.opensea.io/).
+
+## Before you begin
+
+Before you start this tutorial, complete the following items:
+
+- Set up a cryptocurrency wallet
+- To deploy this contract on testnets, ensure the deployer account has testnet LINK. Use the [LINK faucet](https://faucets.chain.link/) to retrieve testnet LINK.
+- Open a new [Remix](https://remix.ethereum.org/) project.
+
+## Run steps
+
+1. Open the `2_complete.sol` contract in Remix and rename the contract to `dynNFT.sol`.
```

Comment:
<img width="231" alt="Screen Shot 2023-08-24 at 4 13 06 PM" src="https://github.com/smartcontractkit/documentation/assets/5498502/42789a9b-d8ac-4b2b-a179-3d9f85756d51">

This CSS issue has come up on a few of my PRs. Two possible solutions:

- Works sometimes, a bit finicky: Make sure this bit of HTML is all in one line, using prettier-ignore if necessary
- Works more reliably: Use the `CodeSample` component with `showButtonOnly` parameter

Reply:
CodeSample works only w our local files; checked over this. Resolved for now. may use prettier-ignore